### ISSUE TITLE
Adding a message collector

### ISSUE THREAD

**Issue by Blizzard098** [Dragory/modmailbot#357]
Hi,

I wanted to add a message collector to the thread when someone sends a DM. What I'm trying to accomplish is something like:

**-> User DMs the bot**

**-> The bot stores the message and asks them a followup question (i.e. What staff team are you attempting to contact?)**

**They enter some keywords (i.e. Support, General, Management) and the bot pings the appropriate group**

The pinging issue I have no trouble doing. It's the part where I make it so the followup answer (to the question) is not logged in the ticket. Sorry if I'm being unclear haha.

**Comment by k9lego** [comment]
Would this be against the Discord API TOS?

**Comment by Computer-Man497** [comment]
Yeah as a matter of fact, i'm actually working on that. I'll open the PR in a few days!

**Comment by k9lego** [comment]
I like that idea, cool!
